tests/models/whisper/test_modeling_whisper.py

```diff
@@ -2520,17 +2558,17 @@ def test_default_multilingual_transcription_long_form(self):
         # task defaults to transcribe
         sequences = model.generate(input_features, return_timestamps=True)
 
-        transcription = processor.batch_decode(sequences)[0]
+        transcription = processor.batch_decode(sequences, skip_special_tokens=False)[0]
 
-        assert transcription == " मिर्ची में कितने विबिन्द प्रजातियां हैं? मिर्ची में कितने विबिन्द प्रजातियां हैं?"
+        assert transcription == "<|startoftranscript|><|hi|><|transcribe|> मिर्ची में कितने विबिन्द प्रजातियां हैं? मिर्ची में कितने विबिन्द प्रजातियां हैं?<|endoftext|>"
 
         # set task to translate
         sequences = model.generate(input_features, task="translate", return_timestamps=True)
-        transcription = processor.batch_decode(sequences)[0]
+        transcription = processor.batch_decode(sequences, skip_special_tokens=False)[0]
```

Comment:
let's also do add a verification with `skip_special_tokens=True`

Reply:
Waiting to have [this](https://github.com/huggingface/transformers/pull/34135#issuecomment-2410846832) decided to eventually remove the special tokens.

Reply:
decided to remove